src/hera.cpp

```diff
@@ -134,7 +134,12 @@ vector<uint8_t> evm2wasm_js(vector<uint8_t> const& input) {
   string fileEVM = mktemp_string("/tmp/hera.evm2wasm.evm.XXXXXX");
   string fileWASM = mktemp_string("/tmp/hera.evm2wasm.wasm.XXXXXX");
 
-  if (fileEVM.size() == 0 || fileWASM.size() == 0)
+  if (
+    fileEVM.size() == 0 ||
+    fileWASM.size() == 0 ||
+    unlink(fileEVM.data()) != 0 ||
```

Comment:
Where it was moved from?

I think this is wrong. If removing of the first file fails, the removal of the second one is not even tried.

Reply:
It is a failure case anyway?